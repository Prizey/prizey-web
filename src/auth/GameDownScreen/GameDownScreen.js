import React from 'react'
import { Button, Typography } from '@material-ui/core'
import { withStyles } from '@material-ui/core/styles'
import Layout from 'design/Layout/Layout'

const styles = theme => ({
  pageTitle: {
    marginBottom: theme.spacing.sm,
  },
})

const createGameDownPage = withStyles(styles)(
  ({ classes, currentUser, location }) => (
    <Layout location={location} currentUser={currentUser}>
      <Typography align="center" variant="h2" className={classes.pageTitle}>
        The game is closed for maintanance
      </Typography>

      <Typography align="center" variant="h5">
        <Button
          href="mailto:contact@prizeyapp.com"
          variant="contained"
          color="primary"
          fullWidth
        >
          contact@prizeyapp.com
        </Button>
      </Typography>
    </Layout>
  ),
)

export default createGameDownPage
