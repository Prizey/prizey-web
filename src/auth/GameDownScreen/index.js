import GameDownScreen from './GameDownScreen'

export default GameDownScreen
