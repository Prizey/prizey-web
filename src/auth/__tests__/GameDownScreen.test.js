import React from 'react'
import renderer from 'react-test-renderer'
import GameDownScreen from '../GameDownScreen'

jest.mock('../../design/Layout/RegisterPageView')

it('renders correctly', () => {
  const tree = renderer.create(<GameDownScreen />).toJSON()
  expect(tree).toMatchSnapshot()
})
