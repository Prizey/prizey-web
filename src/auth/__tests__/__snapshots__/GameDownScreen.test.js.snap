// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="Component-root-9"
>
  <div
    className="Component-header-7"
  />
  <div
    className="Component-container-4"
  >
    <a
      aria-current="page"
      href="/"
      onClick={[Function]}
      style={
        Object {
          "textDecoration": "none",
        }
      }
    >
      <img
        alt="Prizey"
        className="Component-logoImage-8"
        src="/logo.png"
      />
    </a>
    <div
      className="Component-caption-2"
    />
    <div
      className="Component-children-3"
    >
      <h2
        className="MuiTypography-root-10 MuiTypography-h2-23 MuiTypography-alignCenter-33 Component-pageTitle-1"
      >
        The game is closed to maintanance
      </h2>
      <h5
        className="MuiTypography-root-10 MuiTypography-h5-26 MuiTypography-alignCenter-33"
      >
        <a
          className="MuiButtonBase-root-72 MuiButton-root-46 MuiButton-contained-57 MuiButton-containedPrimary-58 MuiButton-raised-60 MuiButton-raisedPrimary-61 MuiButton-fullWidth-71"
          href="mailto:contact@prizeyapp.com"
          onBlur={[Function]}
          onContextMenu={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          role="button"
          tabIndex="0"
        >
          <span
            className="MuiButton-label-47"
          >
            contact@prizeyapp.com
          </span>
          <span
            className="MuiTouchRipple-root-75"
          />
        </a>
      </h5>
    </div>
    <div
      className="Component-footer-5"
    >
      <a
        className="Component-footerLink-6"
        href="https://app.termly.io/document/terms-and-conditions/4de5f3bb-a333-47bb-9daf-f82f7d4e1cfb"
        rel="noopener noreferrer"
        target="_blank"
      >
        <span
          className="MuiTypography-root-10 MuiTypography-caption-20 MuiTypography-colorTextSecondary-43 MuiTypography-alignCenter-33"
        >
          TERMS
        </span>
      </a>
      <a
        className="Component-footerLink-6"
        href="https://app.termly.io/document/privacy-policy/8192f107-a2dc-4548-bf83-97907c6d83c3"
        rel="noopener noreferrer"
        target="_blank"
      >
        <span
          className="MuiTypography-root-10 MuiTypography-caption-20 MuiTypography-colorTextSecondary-43 MuiTypography-alignCenter-33"
        >
          PRIVACY
        </span>
      </a>
      <a
        className="Component-footerLink-6"
        href="https://termly.io/en/"
        rel="noopener noreferrer"
        target="_blank"
      >
        <span
          className="MuiTypography-root-10 MuiTypography-caption-20 MuiTypography-colorTextSecondary-43 MuiTypography-alignCenter-33"
        >
          FAIRNESS
        </span>
      </a>
      <a
        className="Component-footerLink-6"
        href="/all-prizes"
        rel="noopener noreferrer"
        target="_blank"
      >
        <span
          className="MuiTypography-root-10 MuiTypography-caption-20 MuiTypography-colorTextSecondary-43 MuiTypography-alignCenter-33"
        >
          ALL PRIZES
        </span>
      </a>
    </div>
  </div>
</div>
`;
